avaframe/com1DFAPy/DFAfunctionsCython.pyx

```diff
@@ -909,14 +913,26 @@ def computeGradC(cfg, particles, header, double[:, :] Nx, double[:, :] Ny,
   GHZ : 1D numpy array
       z component of the lateral force
   """
+  # get all inputs
+  # configuration parameters
   cdef double rho = cfg.getfloat('rho')
   cdef double minRKern = cfg.getfloat('minRKern')
   cdef double velMagMin = cfg.getfloat('velMagMin')
   cdef double gravAcc = cfg.getfloat('gravAcc')
   cdef int interpOption = cfg.getint('interpOption')
   cdef int SPHoption = cfg.getint('sphOption')
-  cdef double gravAcc3
-  cdef double csz = header.cellsize
+  # SPH grid information and neighbour information
+  cdef double cszSPH = headerSPH.cellsize
+  cdef int nrowsSPH = headerSPH.nrows
+  cdef int ncolsSPH = headerSPH.ncols
+  cdef int[:] indPartInCell = particles['indPartInCell']
+  cdef int[:] partInCell = particles['partInCell']
+  # SPH kernel
+  # use "spiky" kernel: w = (rKernel - r)**3 * 10/(pi*rKernel**5)
+  cdef double rKernel = cszSPH
+  cdef double facKernel = 10.0 / (math.pi * rKernel * rKernel * rKernel * rKernel * rKernel)
```

Comment:
Is this wanted?
```suggestion
  cdef double facKernel = 10.0 / (math.pi * ( 5 * rKernel))
```

Reply:
its a multiplication, not an addition...